internal/util/addmetricsannotation/addmetricsannotation.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package addmetricsannotation
```

Comment:
I don't think `metric` is the right term for what we are doing here. This pkg provides utility methods to record kpt usage or record kpt actions. Metrics part will be realized at the backend by aggregating the action data (or usage data).

Also, use noun for package name is recommended practice. Some pkg names that comes to mind:

```Go
package usage

type Tracker {

}

// The calls in the client code looks like:
usage.Tracker{}.TrackAction("live")

or

type Recorder {

}

usage..Recorder{}.RecordAction("live", ...)

```

Reply:
Agree. Updated.